src/scenes/levelend-scene/levelend-scene.ts

```diff
@@ -113,12 +94,7 @@ export class LevelEndScene {
     }
   };
 
-  draw(deltaTime?: number) {
-    // No background drawing here as it's handled in HTML
-    // Draw buttons
-    this.closeButton.draw();
-    this.retryButton.draw();
-    this.isLastLevel && this.nextButton.draw();
+  draw() {
```

Comment:
This is mostly used for Canvas, if not needed anymore since it is in a HTML approach why not fully clean out the class by removing all unnecessary codes used on previous approach?

Reply:
done here. pls check @bernhardccodev